extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -77,6 +77,16 @@ class AmpStickyAd extends AMP.BaseElement {
     this.removeOnScrollListener_();
   }
 
+  /** @override */
+  collapsedCallback(child) {
+    this.vsync_.mutate(() => {
+      //TODO(zhouyx): Need deregister ad here,
+      //and move updatePadding to detachedCallback();
+      removeElement(this.element);
```

Comment:
Good point! I changed my code. 
Also changed code in `unlayoutCallback()` and `layoutCallback()` to update `borderBottom`